src/test/java/org/cactoos/map/MapOfTest.java

```diff
@@ -108,4 +108,15 @@ public void createsMapWithFunctions() {
         );
     }
 
+    @Test
```

Comment:
@SharplEr Can you also add 2 more tests? One where there is a null or an empty string in one of the values and one where you have complex types such as a List or another Map in one of the keys or values?

Reply:
Done.